frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -25,51 +24,93 @@ const UserPhoto = styled(IconButton)({
   "&:active": {
     background: "#2938a5",
   },
+  // avatar on header
+  ".MuiAvatar-root": {
+    height: "32px",
+    width: "32px",
+    fontSize: "14px",
+    lineHeight: "18px",
+  },
 });
 
+// header and menu avatar
 const Avatar = styled(MuiAvatar)({
-  backgroundColor: "#dce7f4",
-  height: "28px",
-  width: "28px",
-  border: "2px solid #f6faff",
-  borderRadius: "50%",
-});
-
-const Initials = styled(Typography)({
-  color: "#0d1030",
-  opacity: 0.6,
-  fontSize: "14px",
+  backgroundColor: "#727FE1",
+  color: "#FFFFFF",
   fontWeight: 500,
 });
 
 const Paper = styled(MuiPaper)({
-  width: "266px",
-  border: "1px solid #e2e2e6",
+  width: "242px",
+  border: "1px solid #E7E7EA",
   boxShadow: "0px 5px 15px rgba(53, 72, 212, 0.2)",
 });
 
 const Popper = styled(MuiPopper)({
-  padding: "0 12px",
-  marginLeft: "12px",
+  padding: "0 6px",
+  marginLeft: "6px",
+  zIndex: 1101,
```

Comment:
since this commit https://github.com/lyft/clutch/pull/760/files, the avatar popper has been hidden behind the appbar. so added the [appbar zindex + 1](https://material-ui.com/customization/z-index/)